src/components/ui/experience.tsx

```
@@ -133,12 +141,21 @@
     setWasValidated(false);
   };
 
-  const handleDelete = () => {
-    if (editingExperience) {
-      setExperienceList((prev) =>
-        prev.filter((exp) => exp.id !== editingExperience.id)
-      );
-      setIsModalOpen(false);
+  const handleDelete = async () => {
+    if (editingExperience && editingExperience.id) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)